src/character.cpp

```diff
@@ -3326,10 +3327,16 @@ bool Character::meets_skill_requirements( const construction &con ) const
 
 bool Character::meets_stat_requirements( const item &it ) const
 {
-    return get_str() >= it.get_min_str() &&
-           get_dex() >= it.type->min_dex &&
-           get_int() >= it.type->min_int &&
-           get_per() >= it.type->min_per;
+    if( it.has_flag( flag_STR_DRAW ) ) {
+        return get_dex() >= it.type->min_dex &&
+               get_int() >= it.type->min_int &&
+               get_per() >= it.type->min_per;
+    } else {
+        return get_str() >= it.get_min_str() &&
+               get_dex() >= it.type->min_dex &&
+               get_int() >= it.type->min_int &&
+               get_per() >= it.type->min_per;
+    }
```

Comment:
More of a nitpick, but if and else blocks are almost identical, and can be shortened to
```suggestion
    return ( it.has_flag( flag_STR_DRAW ) || get_str() >= it.get_min_str() ) &&
           get_dex() >= it.type->min_dex &&
           get_int() >= it.type->min_int &&
           get_per() >= it.type->min_per;
```